app/_src/mesh/features/opa.md

```diff
@@ -18,9 +18,11 @@ When `OPAPolicy` is applied, the control plane configures:
 To apply a policy with OPA:
 
 - Specify the group of data plane proxies to apply the policy to with the `selectors` property.
-- Provide the list of policies with the `conf` property. Policies are defined in the [Rego language](https://www.openpolicyagent.org/docs/latest/policy-language/).
+- Provide a policy<a href="#footnote-1"><sup id="note-return-1">1</sup></a> with the `conf` property. Policies are defined in the [Rego language](https://www.openpolicyagent.org/docs/latest/policy-language/).
```

Comment:
```suggestion
- Provide a policy with the `conf` property. Policies are defined in the [Rego language](https://www.openpolicyagent.org/docs/latest/policy-language/).
{:.note}
> **Note:** As of version 2.1.x, you cannot currently apply multiple OPA policies. This limitation will be addressed in the future.
```
I tried to make this into a note instead of a footnote just so that the info would be right under the bullet point.

Reply:
> As of version 2.1.x

I'm not a native speaker but I think this is incorrect. As I mentioned in the PR description this never worked so **all versions of KM** are affected, but this indicates that only `>=2.1.0` are affected right?